src/Microsoft.TestPlatform.Common/ExtensionFramework/Utilities/TestExtensions.cs

```diff
@@ -134,32 +134,33 @@ public class TestExtensions
         /// <returns> The test extensions defined the extension assembly if it is already discovered. null if not.</returns>
         internal TestExtensions GetExtensionsDiscoveredFromAssembly(string extensionAssembly)
         {
-            var testExtensions = new TestExtensions();
-
-            testExtensions.TestDiscoverers =
+            var testExtensions = new TestExtensions
```

Comment:
Using object initializers would make this very hard to debug because the whole initializer will fail with the exception without properly indicating where it failed, please revert.